changelog.txt

```diff
@@ -1,19 +1,23 @@
+Version 1.4.0 - 26th December, 2024
```

Comment:
This should be v1.3.4 since no new component is introduced. @jaieds

Reply:
Okay. I will update it to `v1.3.4`.